gmt/views/general.py

```diff
@@ -85,9 +85,11 @@ def news():
     if not posts:
         posts = get_news(choice="BBC")
 
+    random.shuffle(posts)
+
     return render_template(
```

Comment:
Sorry I'm a bit busy so I can't do this but can you add an if statement for the hostname for this? Like that it's only viewable on the dev deployment and maybe even with a password? Otherwise people will just scrape this content which isn't the intention

Reply:
I limited the number of posts to 12 so they cant scrape the entire db and if someone really wants to scrape them, I am honestly fine with it, as long as it doesn't harm us